WebGoatCore/Exceptions/WebGoatCreditCardNotFound.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace WebGoatCore.Exceptions
+{
+    public class WebGoatCreditCardNotFoundException : AbstractWebGoatException

```

Comment:
Please rename file to match class name (missing Exception in file name)

Reply:
Done.